frontend/src/components/Documentation.vue

```diff
@@ -131,8 +131,8 @@
           <p>Similar to the 2D SVG maps, the 3D graphs are interactive. Clicking a node on the 3D grap, be it a metabolite, reaction, or gene, the information panel on the left sidebar will be updated for the clicked node. Additonal information of the selected node can be accessed by clicking the button "GEM Browser" under information panel on the left sidebar.</p>
 
           <h5 id="Data-overlay" class="is-size-5">Data overlay</h5>
-          <div class="columns m-0">
-            <div class="column p-0 is-8 content">
+          <div class="columns is-8">
```

Comment:
Is the indentation of the paragraph "Data overlay" made by purpose or a side affect of increasing the gap between columns, see https://www.dropbox.com/s/mdk1pv78lisahz5/Screenshot%202021-03-16%20at%2022.10.29.png?dl=0

Reply:
Great that you spotted this @nanjiangshu. That extra indentation is a mistake, definitely not on purpose, but it might have to do will the changes that have increased gap between the columns.

Reply:
How does it look now after https://github.com/MetabolicAtlas/MetabolicAtlas/pull/542/commits/9dd8bd277f77d923e0237ad955bafbeb48db5d6d?

Reply:
Looks great now! I will merge the branch.